### PR TITLE
ZEPPELIN-1440 Notebook clone: prefix name with "Copy of" and end with count

### PR DESCRIPTION
**Pull request by r-kamath**
### What is this PR for?
While cloning a notebook create a new name using the current notebook name by prefixing "Copy of" and end with count

e.g:
name: test
pre-filled clone name: Copy of test 1

### What type of PR is it?
Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1440

### How should this be tested?
Create a new book and clone on ui or see unit test in notename.js

### Screenshots (if appropriate)
Before
before

After
after

### Questions:
* Does the licenses files need update? na
* Is there breaking changes for older versions? na
* Does this needs documentation? na

### CODE REVIEW

**Comment by felixcheung** [comment]
I think it might be easier if we don't prefix "Copy of"? We might need to localize or make this localizable at some point

**Comment by r-kamath** [comment]
@felixcheung I think either we remove "Copy of" or make it localizable. Will wait comments from others before making the choice.

**Comment by AhyoungRyu** [comment]
@r-kamath Thanks for the improvement. It's better than just "Untitled Note #" . 

But regarding `Copy of`, I agree with @felixcheung. If an user tries to clone a notebook in `this/is`, then `is 1` note will be created under `Copy of this` folder. It's a bit awkward in my point of view.
  

So just numbering would be enough I guess. What do you think? :)

**Comment by corneadoug** [comment]
Except for the folder issue that @AhyoungRyu reported, I found another bug where the number do not increment:
* Make a clone
* Go back to the original note
* Make another clone, the number is still 1 even though 1 already exist

**Comment by r-kamath** [comment]
@felixcheung removed "Copy of" prefix.
@AhyoungRyu fixed folder related issue.
@corneadoug fixed sequence bug.

**Comment by AhyoungRyu** [comment]
@r-kamath Thanks! Tested again and looks good 👍

**Comment by corneadoug** [comment]
@r-kamath Could you just rebase once more? I will make a final check after

**Comment by prabhjyotsingh** [comment]
Tested, LGTM

File: zeppelin-web/test/spec/controllers/notename.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+describe('Controller: NotenameCtrl', function() {
+  beforeEach(module('zeppelinWebApp'));
+
+  var scope;
+  var ctrl;
+  var notebookList;
+
+  beforeEach(inject(function($injector) {
+    notebookList = $injector.get('notebookListDataFactory');
```
Comment (corneadoug) [inline zeppelin-web/test/spec/controllers/notename.js]: Why 2 beforeEach functions?
This could be moved to the other one

**Comment by corneadoug** [comment]
Except for my comment about the test, LGTM.
CI failure is irrelevant

**Comment by prabhjyotsingh** [comment]
@r-kamath do you want to try and merge this ?
[This](https://cwiki.apache.org/confluence/display/ZEPPELIN/Committer+Guide) will help you.

**Comment by r-kamath** [comment]
@corneadoug removed the redundant beforeEach b8b4f24
Will merge if there are no more discussion

**Comment by corneadoug** [comment]
@r-kamath If you have trouble merging, tell me

**Pull request closed without merging**